library/core/src/net/ip_addr.rs

```diff
@@ -1941,6 +1989,35 @@ impl Ipv6Addr {
     pub const fn octets(&self) -> [u8; 16] {
         self.octets
     }
+
+    /// Creates an `Ipv6Addr` from a sixteen element byte array.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// #![feature(ip_from)]
+    /// use std::net::Ipv6Addr;
+    ///
+    /// let addr = Ipv6Addr::from_octets([
+    ///     25u8, 24u8, 23u8, 22u8, 21u8, 20u8, 19u8, 18u8,
+    ///     17u8, 16u8, 15u8, 14u8, 13u8, 12u8, 11u8, 10u8,
+    /// ]);
+    /// assert_eq!(
+    ///     Ipv6Addr::new(
+    ///         0x1918, 0x1716,
+    ///         0x1514, 0x1312,
+    ///         0x1110, 0x0f0e,
+    ///         0x0d0c, 0x0b0a
+    ///     ),
```

Comment:
Tiny nit - for all the examples it might be nicer to use hex for both `addr` and the `assert!`, since it's easier to see that `0x19, 0x18` turns into `0x1918` than it is to see that `25, 24` becomes `0x1918`.

r=me with that

Reply:
agreed! It used decimal because I copied it from the existing examples for the `From` impls. I've changed them all to hex.

Reply:
Thanks! Would be best if you could squash then I'll approve it

Reply:
Oh nevermind don't squash, those are two separate things. LGTM.